tests/test_update_config.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+from pytest import fixture as pytest_fixture
+
+from sanic import Sanic
+
+
+@pytest_fixture
+def sanic_app():
+    return Sanic()
+
+
+@pytest_fixture(
+    params=[
+        {"A": 1},
+        type("C", (), {"A": 1}),
+        Path(__file__).parent / "static/app_conf.py",
```

Comment:
Is there a way to run this test (load config from file) only when test_load_module_from_file_location test succed ?  
(I have not dig into pytest so much).

Reply:
I found the solution: pytest-dependency.